tests/scripts/arm64_post_build.py

```diff
@@ -280,10 +323,10 @@ def main(args):
            build_type, 
            scenario, 
            core_root, 
-           g_test_url]
+           test_location]
 
    log(" ".join(args))
-   proc = subprocess.Popen(args, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+   proc = subprocess.Popen(args)
    std_out, std_err = proc.communicate()
```

Comment:
This is actually an error, the std_out and std_err here will be None thank you for catching this.